static/app/components/avatar/baseAvatar.tsx

```diff
@@ -205,11 +205,12 @@ class BaseAvatar extends Component<Props, State> {
 
   renderLetterAvatar() {
     const {title, letterId, round, suggested} = this.props;
+    const modifiedTitle = title?.charAt(0) === '[' ? '?' : title;
```

Comment:
Hmm, would have a strict check for `[Filtered]` be better? I'm not sure if we have other magic strings to check for

Reply:
yup that makes sense to me